### ISSUE TITLE
Disallow creation of multiple socks on same port

### ISSUE THREAD

**Issue by sanderfoobar** [knaccc/i2p-zero#21]
1. Run `socks.create 31338` multiple times.
2. Run `all.list`.

```
        {
            "port": "31338",
            "state": "opening",
            "type": "socks"
        },
        {
            "port": "31338",
            "state": "opening",
            "type": "socks"
        },
        {
            "port": "31338",
            "state": "opening",
            "type": "socks"
        }
```

**Comment by knaccc** [comment]
Good point, will fix

**Comment by knaccc** [comment]
Fixed in v1.10
